### PR TITLE
[Qwen]Fix performance issue with dynamic causal mask

### PR DESCRIPTION
**Pull request by zhangYiIntel**
# What does this PR do?

Fixes Qwen performance issue with dynamic causal mask generation

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: optimum/exporters/openvino/model_patcher.py [hunk 1]
```diff
@@ -444,9 +444,10 @@ def _qwen_attention_forward(
             value = value.permute(0, 2, 1, 3)
 
         if not self.use_cache_quantization and SUPPORT_SDPA:
-            causal_mask = registered_causal_mask[:, :, key.size(-2) - query.size(-2) : key.size(-2), : key.size(-2)]
+            # For performance, using constant tril to generate causal_mask
+            causal_mask = self.bias[:, :, key.size(-2) - query.size(-2) : key.size(-2), : key.size(-2)]
```
Comment (eaidova) [inline optimum/exporters/openvino/model_patcher.py:448]: did you check is it work with the latest qwen version?

Comment (zhangYiIntel) [reply optimum/exporters/openvino/model_patcher.py:448]: Yes, I test the patch with the latest Qwen-7b-chat model (commit `93a65d34827a3cc269b727e67004743b723e2f83`). It works well. The usage of SDPA remains the same between commit `2abd8e5777bb4ce9c8ab4be7dbbd0fe4526db78d` and `93a65d34827a3cc269b727e67004743b723e2f83`

**Comment by eaidova** [comment]
@echarlaix  could you please take a look?

**Review by eaidova — APPROVED** [review]

**Comment by echarlaix** [comment]
To fix the code style test you can do the following : 
```
pip install .[quality]
make style
```

**Review by echarlaix — APPROVED** [review]

**Pull request merged**